packages/dcos-ui-update-service/extra/dcos-ui-update-service.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=DC/OS UI Update Service: dcos ui update service for updating the ui
+
+[Service]
+Restart=always
+RestartSec=5
+
+PermissionsStartOnly=True
+
+User=dcos_ui_update_service
+
+SupplementaryGroups=dcos_adminrouter
+
+EnvironmentFile=/opt/mesosphere/environment
+
+ExecStartPre=/bin/bash -c "mkdir -p /opt/mesosphere/active/dcos-ui-update-service/versions"
```

Comment:
Also, these commands should be probably moved into the bootstrap package.

Reply:
Do we have any example of such bootstrap? Or do you mean `build`? Having it in `build` is impossible because at that stage we don't have the user yet. 
We've found that to be a pattern: https://github.com/dcos/dcos/blob/0f1b44e591e23c2bb8c0b2105de92f6510449bb0/packages/telegraf/extra/dcos-telegraf.service

Reply:
Apparently, when you use `state_directory: true` in your `pkgpanda` build file the state directory will get created for you - https://github.com/mesosphere/dcos-enterprise/blob/e49a666d7aa9a4aff62b599a2cbf699691f09680/packages/bootstrap/extra/dcos_internal_utils/bootstrap.py#L1802-L1805

So you may not need to create any `bootstrap` procedure for your component.

TIL 🤷‍♂️